Makefile

```diff
@@ -357,14 +357,14 @@ docker-nuke:
 	$(MAKE) init-contracts
 
 docker-remote-watcher:
-	docker-compose exec watcher /watcher_entrypoint bin/watcher remote_console
+	docker exec -ti watcher /app/bin/child_chain remote
 
 docker-remote-watcher_info:
-	docker-compose exec watcher_info /watcher_info_entrypoint bin/watcher_info remote_console
+	docker exec -ti watcher_info /app/bin/child_chain remote
 
 docker-remote-childchain:
-	docker-compose exec childchain /child_chain_entrypoint bin/child_chain remote_console
-
+	docker exec -ti childchain /app/bin/child_chain remote
```

Comment:
Got an error running `make docker-remote-childchain`:
```shell
$ make docker-remote-childchain
OCI runtime exec failed: exec failed: container_linux.go:349: starting container process caused "exec: \"/app/bin/child_chain\": stat /app/bin/child_chain: no such file or directory": unknown
make: *** [docker-remote-watcher] Error 126
```

Reply:
hmmm interesting.
`make docker-build-cluster`
```
elixir-omg  inomurko/bump_elixir_geth $ make docker-remote-childchain
Erlang/OTP 22 [erts-10.7] [source] [64-bit] [smp:3:3] [ds:3:3:10] [async-threads:1]

Interactive Elixir (1.10.2) - press Ctrl+C to exit (type h() ENTER for help)
iex(child_chain@127.0.0.1)1>
```

```
docker --version
Docker version 19.03.8, build afacb8b
```
```
docker-compose version
docker-compose version 1.25.4, build 8d51620a
docker-py version: 4.1.0
CPython version: 3.7.5
OpenSSL version: OpenSSL 1.1.1d  10 Sep 2019
```

Are you on these versions? I'm also on Catalina 10.15.4

Reply:
Surprisingly exactly the same. Lemme try run again with your latest push.

```
$ docker --version
Docker version 19.03.8, build afacb8b

$ docker-compose version
docker-compose version 1.25.4, build 8d51620a
docker-py version: 4.1.0
CPython version: 3.7.5
OpenSSL version: OpenSSL 1.1.1d  10 Sep 2019

$ sw_vers -productVersion
10.15.4
```

Reply:
Pulled the latest commit. Nuked everything including docker, build caches, mix caches. Works now!